### DISCUSSION TITLE
How to change the bitrate when re-encoding?

### DISCUSSION THREAD

**Discussion by kelzan** [yermak/AudioBookConverter#531] [Q&A]
It seems like I must be missing something obvious, but when I convert an mp3 to m4b, it keeps whatever bitrate the original mp3 was, and ignores any settings that I have selected in the _Quality_ tab. For example, if I start with a 128kb/s mp3, and want to encode it into a 64kb/s, I still end up with a similar sized m4b at 128kb/s, even though I've specified 64kb/s.

I've set _Force re-encoding_ to _Always_, but that doesn't seem to have any effect. Any suggestions on how to get the tool to honor the settings on the _Quality_ tab when it does the conversion?

**Comment by yermak** [comment]
Hi,
need few details to advise.
Did you celect CBR? (Constant bitrate) it could be that you using VBR - which (depending on quality) may give you similar outcome to original file.

Operating system, version of AudiobookConverter might be useful for further investigation.
